produce/README.md

```diff
@@ -173,6 +177,28 @@ Get a list of all available options using
     --vpn-conf           Disable VPN Configuration
 ```
 
+## Creating Apple Pay merchants and associating them with an App ID
+
+If you want to create a new Apple Pay Merchant Identifier:
+
+    fastlane produce merchant -o merchant.com.example.production -r "Example Merchant Production"
```

Comment:
Can we please add these examples to the Fastfile example bellow if possible. I am a bit confused on how to implement the new merchant params. Thanks!